crates/pgt_schema_cache/src/queries/schemas.sql

```diff
@@ -1,7 +1,35 @@
 select
   n.oid :: int8 as "id!",
   n.nspname as name,
-  u.rolname as "owner!"
+  u.rolname as "owner!",
+  obj_description(n.oid, 'pg_namespace') as "comment",
+
+  coalesce((
+    select array_agg(grantee::regrole::text)
+    from aclexplode(n.nspacl)
+    where privilege_type = 'USAGE'
+    and grantee::regrole::text <> ''
+    and grantee::regrole::text <> '-'
```

Comment:
i found some empty roles in my local supabase database.

`select nspacl from pg_namespace where nspname = 'public'` yielded:

{postgres=UC/postgres,anon=U/postgres,authenticated=U/postgres,service_role=U/postgres,`=UC/postgres`,airbyte=U/postgres,supabase_auth_admin=U/postgres}